lib/webauto/web4cucumber.rb

```diff
@@ -136,20 +136,13 @@ def browser
         end
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
-        #https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
-        chrome_caps[:acceptInsecureCerts] = true
-        if Integer === @scroll_strategy
-          chrome_caps[:element_scroll_behavior] = @scroll_strategy
-        end
```

Comment:
I don't remember details but I remember that we needed this scroll strategy to find some elements that were otherwise hidden behind UI menus. Maybe not needed with latest UI. idk, but just FYI in case you start seeing strange errors with elements not being visible.

Reply:
yeah, we added this code to scroll element into view before taking action on it @liangxia  Can we achieve the same goal with new method?

Reply:
@akostadinov  Nice to see you again. Thank you for the remind.
We've compared the test results in master branch and devel branch. Total 230 test cases, and we run the same test on four different clusters on both master branch and devel branch, we only got 2, 2, 4, 3 failures for devel branch in each test run, while 4, 3, 3, 3 failures in master branch. So I think it's fine to remove it.

Reply:
I think we also had a custom scroll code to try to scroll when element overlap was detected. So maybe it works :)